### ISSUE TITLE
[Feature] Ability to listen on a Unix socket rather than TCP

### ISSUE THREAD

**Issue by evitalis** [documize/community#271]
This would help eliminate some overhead when running with a reverse proxy. Another configuration option might be needed though to accommodate this unless the port option gets repurposed. In addition this would also help remove some configuration with the firewall and avoid accidental exposure of the port if it somehow listens on something other than localhost.

**Comment by HarveyKandola** [comment]
To be clear, serve HTTP over Unix domain sockets, right?

**Comment by evitalis** [comment]
Yes. A .sock file should be created under /run for the application to listen on instead of a TCP socket.

**Comment by HarveyKandola** [comment]
That should be doable where the config file specifies the SOCK config.

**Comment by evitalis** [comment]
@HarveyKandola I would suggest it be in the format of `unix:///PATH.sock` where PATH is the full filesystem path to the socket file. This would make it consistent with other applications that work in this way.
